pkgs/tools/misc/zoxide/default.nix

```diff
@@ -3,9 +3,17 @@
 , rustPlatform
 , withFzf ? true
 , fzf
+
```

Comment:
this new can be removed :)

Reply:
It's cosmetic, yes, but I prefer the extra separation (even though there is already a comment describing the separation).